modules/claims_api/app/controllers/claims_api/v1/forms/power_of_attorney_controller.rb

```diff
@@ -65,6 +65,7 @@ def submit_form_2122 # rubocop:disable Metrics/MethodLength
         #
         # @return [JSON] Claim record
         def upload
+          validate_document_provided
           validate_documents_content_type
```

Comment:
I am probably missing something, but where is this method implemented?

Reply:
[It's here](https://github.com/department-of-veterans-affairs/vets-api/blob/92496b92264bcf23da1f262160c60094752a68a9/modules/claims_api/app/controllers/concerns/claims_api/document_validations.rb#L21-#L25)